### DISCUSSION TITLE
How to see on Github how  much of storage has left in my account.

### DISCUSSION THREAD

**Discussion by VitRod** [community/community#31094] [Other Feature Feedback, Questions, & Ideas] [answered]
I have loaded  several  videos and now i would  like to see how  many  megabites of 500 in my storage have left. Thank you in advance for the  answer.

**Comment by melvincwng** [comment, marked as answer]
Hey @VitRod 👋, to check your storage limits/capacity, you can head over to this link over here: https://github.com/settings/billing

Once at that link, head over to 'Storage for Actions and Packages' or 'Git LFS Data' to check your storage limits.

Hope this answers your question 😊

**Reply by VitRod** [reply]
Thank you very much. You really helped me a lot.

**Reply by melvincwng** [reply]
@VitRod Pleasure in helping! Glad to have helped you 😊

**Reply by DirtyLeon** [reply]
@melvincwng Hi. Mine doesn't tell me how much storage capacity I have left for my LFS storage, instead it just show me billable usage in dollars. Is that normal?
I was expecting to see something like "You are using 14.7GB out of 50GB storage from your purchased data pack" or something like that.
![image](https://github.com/user-attachments/assets/6bab5bf2-f890-42a0-9372-36d50122862f)

**Reply by HugoCortell** [reply]
> Hey @VitRod 👋, to check your storage limits/capacity, you can head over to this link over here: https://github.com/settings/billing
> 
> Once at that link, head over to 'Storage for Actions and Packages' or 'Git LFS Data' to check your storage limits.
> 
> Hope this answers your question 😊

Is this still correct? It shows literally zero for me, despite having multiple repos (quite a few heavy in binary files).
